src/oiiotool/oiiotool.cpp

```diff
@@ -6137,6 +6160,11 @@ Oiiotool::getargs(int argc, char* argv[])
     ap.arg("--threads %d:N")
       .help("Number of threads (default 0 == #cores)")
       .OTACTION(set_threads);
+    ap.arg("--cuda")
```

Comment:
perhaps more generically forward looking `--gpu=cuda`? allowing for `--gpu=metal` in the future without an explosion of arguments.

Reply:
Yes, you're right.

I also had a question in my mind about the "gpu" terminology. What I really am trying to express is "compute device", but I'm using gpu as a very short stand-in unless somebody can suggest something better.

Reply:
Updated, with change to --gpu.
For now, it does the same thing as before, but in the future we'll expand it to have a way to choose what kind of device to use if there is more than supported and one available.

Reply:
We'll build this all up step by step.